lib/protobuf/rpc/service.rb

```diff
@@ -103,8 +103,9 @@ class << self
       # This methods is only used by the generated service definitions
       # and not useful for user code.
       #
-      def self.rpc(method, request_type, response_type)
-        rpcs[method] = RpcMethod.new(method, request_type, response_type)
+      def self.rpc(method, request_type, response_type, options = {})
+        name = options[:name] || method
```

Comment:
Because of old code that goes to look up `service.rpcs[old_name]` will fail with